src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -592,6 +592,7 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Shutdown() {
   // to prevent it from unloading while in use
   std::lock_guard<std::mutex> guard(module_id_to_info_map_lock_);
 
+  // rejit_handler->Shutdown();
```

Comment:
nit: Can we delete this line?

Reply:
I re-enabled the shutdown process in eb1e7ab